plugins/gatsby-theme-iterative-docs/src/utils/shared/sidebar.js

```diff
@@ -23,12 +23,24 @@
 const { titleCase } = require('title-case')
 const sidebar = require('../../../sidebar')
 
+const uppercaseKeywords = ['dvc', 'cml', 'api', 'mlem', 'ml', 'ldb']
 const PATH_ROOT = '/doc'
 const FILE_ROOT = '/docs/'
 const FILE_EXTENSION = '.md'
 
-function dvcTitleCase(slug) {
-  return titleCase(slug.replace(/dvc/g, 'DVC').replace(/-/g, ' '))
+function uppercaseSlugKeywords(slug) {
+  let newSlug = slug
+  uppercaseKeywords.forEach(word => {
```

Comment:
can we optimize it a bit please? combine into one regex? `regex = [words].join("|")` (this is not an actual lang, just to communicate the idea). Also, pre-compile the regex (not sure is JS has a cache of complied regexps - need to check).

Reply:
Good idea! Forgot that we can easily just use one regex since `replace` has different options for the second argument. 

I also moved the regex outside of the function so that it isn't recompiled on every function run :)